Lib/runpy.py

```diff
@@ -245,14 +258,19 @@ def _get_code_from_file(run_name, fname):
     return code, fname
 
 def run_path(path_name, init_globals=None, run_name=None):
```

Comment:
hm, the docs get the name of the first param wrong. I'll create a separate PR to fix that.